morebits.js

```diff
@@ -4001,6 +4001,71 @@ Morebits.wikitext.page.prototype = {
 		}
 	},
 
+	/**
+	 * Smartly insert a tag atop page text but after specified templates,
+	 * such as hatnotes, short description, or deletion and protection templates.
+	 * Notably, does *not* insert a newline after the tag
+	 *
+	 * @param {string} tag - The tag to be inserted
+	 * @param {string|array} regex - Templates after which to insert tag,
+	 * given as either as a string or an array to be joined by pipes
```

Comment:
```suggestion
	 * @param {string|string[]} regex - Templates after which to insert tag,
	 * given as either as a string or an array to be joined by pipes
```
Also, this should clarify that the strings can be regexes too.

Reply:
Didn't know about `string[]`!